modules/Microsoft.SignalRService/webPubSub/.test/full.parameters.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Should we rename this to parameters.json?

Reply:
As mentioned in one of the other PRs it's very inconsistent and as long as the reference in Azure DevOps pipeline definition is correct, this should be fine. GitHub actions discover all files with .json anyway and soon we are switching to bicep based parameter tests.

Reply:
Agreed with @itpropro - and changed it anyways ;) This way we're all in agreement :)